### PR TITLE
Support Animated Cape Sprites

### PR DESCRIPTION
**Pull request by james090500**
Adds support for animated cape sprites

### CODE REVIEW

File: src/process.ts [hunk 1]
```diff
@@ -164,6 +164,8 @@ function computeCapeScale(image: TextureSource): number {
 	} else if (image.width * 11 === image.height * 23) {
 		// 46x22
 		return image.width / 46;
+	} else if ((image.height % (image.width / 2)) == 0) {
```
Comment (Hacksore) [inline src/process.ts:167]: Can you please add a comment explaining what this branch is for?

**Review by Hacksore — CHANGES_REQUESTED** [review]

File: src/process.ts (hunk 1 above)
Comment (james090500) [reply src/process.ts:167]: I have added requested changes.

**Review by Hacksore — APPROVED** [review]
LGTM but @yushijinhun might have other opinions.

**Comment by james090500** [comment]
When moving the code from https://github.com/bs-community/skinview3d/pull/128/ to here I forgot to include a method for loading a specific frame.

**Comment by james090500** [comment]
Any update on this?

**Comment by james090500** [comment]
Any update on this?

**Comment by Hacksore** [comment]
@yushijinhun can you please review this?

**Comment by Hacksore** [comment]
I believe @james090500 ported this into his lib here.

https://github.com/MinecraftCapes/minecraft-skin-viewer

**Pull request closed without merging**